lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -172,4 +183,20 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
   end
 end
 
+function M.select_temp_directory_change_command_start(dir)
```

Comment:
is there maybe a more pregnant same for this? `select_push_dir_command`/``select_pop_dir_command`